style.css

```diff
@@ -43,10 +43,41 @@
 }
 
 .list_item_active a{
-    color: #e8491d;
+    color:#C1EDFE;
 
 }
 
 .header_nav_list a:hover{
     color: rgb(241, 245, 8);
-}
\ No newline at end of file
+    font-size: 25px;
+
+}
+
+
+ /*Search Style */
+
+.search{
+    justify-content: center; 
+    width:70%;
+    height: 70%;
+    border: 3px solid #C1EDFE;
+    border-radius: 25px;
+    padding: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
+    font-size: 5px;
+}
+
+
+
+
+/*Media Query style */
+
+@media screen and(max-width : 768px){
+
+    /* header style*/
+       
+    } 
+
+
```

Comment:
pay some attention to your code formatting, use prettier correctly

![Peek 2021-03-02 19-15](https://user-images.githubusercontent.com/47992412/109870375-b03f8180-7c72-11eb-9d63-aa7a78526557.gif)